source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/detail/StakeHolderContainer.tsx

```diff
@@ -36,12 +36,14 @@ export const StakeHolderContainer: React.FunctionComponent<IStakeHolderContainer
 
   const allInterestProperties =
     apiInterestHolders?.flatMap(interestHolder => interestHolder.interestHolderProperties) ?? [];
-  const interestProperties = allInterestProperties.filter(ip =>
-    ip.propertyInterestTypes.some(pit => pit?.id !== 'NIP'),
-  );
-  const nonInterestProperties = allInterestProperties.filter(ip =>
-    ip.propertyInterestTypes.some(pit => pit?.id === 'NIP'),
-  );
+  const interestProperties = allInterestProperties.map(ip => {
+    const filteredTypes = ip.propertyInterestTypes.filter(pit => pit?.id !== 'NIP');
+    return { ...ip, propertyInterestTypes: filteredTypes };
```

Comment:
would this not give all properties but some with propertyInterestType set to undefined?

Reply:
mmm, I think you are correct actually, works in the case of a property with both types, but not with just one, need to combine this with previous implementation.